content/backmatter.tex

```diff
@@ -21,7 +21,7 @@ \section*{Incorporating OpenSHMEM into Programs}\label{sec:writing_programs}
 To write a ``Hello World" \openshmem program, the user must:
```

Comment:
Please also add a changelog entry to the effect that the deprecated Fortran interfaces have been removed.

Reply:
Update deprecation annex table “last version supported” to 1.4 for every Fortran row.

Deprecation Annex Section 2.13: Update “deprecated” to “deprecated in v1.4 and removed in v1.5”.

Reply:
Fixed